src/app/route/Checkout/Checkout.container.js

```diff
@@ -241,6 +264,34 @@ export class CheckoutContainer extends PureComponent {
         );
     }
 
+    createUser() {
+        const { createAccount } = this.props;
+        const {
+            email,
+            password,
+            createUser,
+            shippingAddress: {
+                firstname,
+                lastname
+            }
+        } = this.state;
+
+        if (!createUser && !password) return false;
+
+        const options = {
+            customer: {
+                email,
+                firstname,
+                lastname
+            },
+            password
+        };
+
+        return createAccount(options).then(() => {
+            this.setShippingAddress();
```

Comment:
You might not need the () => {} here. Can use `this.setShippingAdress` directly!